aiohttp/cookiejar.py

```diff
@@ -334,9 +332,8 @@ class DummyCookieJar(AbstractCookieJar):
 
     """
 
-    def __init__(self, *,
-                 loop: Optional[asyncio.AbstractEventLoop]=None) -> None:
-        super().__init__(loop=loop)
+    def __init__(self) -> None:
```

Comment:
This `__init__` looks redundant.

Reply:
It initializes `self._loop`.
I agree that `__init__` in `AbstractCookieJar` can be avoided but the change is not backward compatible a little

Reply:
After rethinking I've decided to drop `AbstractCookieJar.__init__`.
Fixed.